src/CarbonAware.WebApi/test/unitTests/TestsBase.cs

```diff
@@ -20,60 +18,63 @@ protected TestsBase()
         this.MockCarbonAwareLogger = new Mock<ILogger<CarbonAwareController>>();
     }
 
-    protected static Mock<IEmissionsAggregator> CreateEmissionsAggregator(List<EmissionsData> data)
+    protected static Mock<IEmissionsHandler> CreateEmissionsHandler(List<EmissionsData> data)
     {
-        var aggregator = new Mock<IEmissionsAggregator>();
-        aggregator.Setup(x => x.GetEmissionsDataAsync(It.IsAny<CarbonAwareParameters>()))
-            .Callback((CarbonAwareParameters parameters) =>
+        var handler = new Mock<IEmissionsHandler>();
+
+        handler.Setup(x => x.GetEmissionsDataAsync(It.IsAny<string[]>(), null, null))
+            .Callback((string[] location, DateTimeOffset? start, DateTimeOffset? end) =>
             {
-                parameters.SetRequiredProperties(PropertyName.MultipleLocations);
-                parameters.Validate();
+                Assert.NotNull(location);
+                Assert.NotZero(location.Length);
             })
             .ReturnsAsync(data);
-        return aggregator;
+        return handler;
     }
 
-    protected static Mock<IEmissionsAggregator> CreateAggregatorWithBestEmissionsData(List<EmissionsData> data)
+    protected static Mock<IEmissionsHandler> CreateHandlerWithBestEmissionsData(List<EmissionsData> data)
     {
-        var aggregator = new Mock<IEmissionsAggregator>();
-        aggregator.Setup(x => x.GetBestEmissionsDataAsync(It.IsAny<CarbonAwareParameters>()))
-            .Callback((CarbonAwareParameters parameters) =>
+        var handler = new Mock<IEmissionsHandler>();
+        handler.Setup(x => x.GetBestEmissionsDataAsync(It.IsAny<string[]>(), null, null))
```

Comment:
We should doc at a minimum why we're passing null parameters back to the handler.

Reply:
I think it may make more sense if I update the null params to `It.IsAny<DateTimeOffset>()` instead, so there is context on the type of optional parameters that can be passed in. Does that seem reasonable?

Reply:
Seems like a great resolution!